src/Adapter/Supplier/QueryHandler/GetSupplierForViewingHandler.php

```diff
@@ -162,11 +165,10 @@ private function getSupplierProducts(Supplier $supplier, LanguageId $languageId)
                     $combinations[$attributeId]['attributes'] = $attribute;
                 }
 
-                $productInfo = Supplier::getProductInformationsBySupplier(
-                    $supplier->id,
-                    $product->id,
-                    0
-                );
+                $productInfo = Supplier::getProductInformationsBySupplier($supplier->id, $product->id);
```

Comment:
I'm not sure but it seems like here you are ignoring every product which don't have a product_supplier with `id_product_atrribute == 0` It's fine for regular products, but I'm afraid this will cause all products with combinations to be ignored
So the page doesn't have a 500 error anymore, but it doesn't display everything it's supposed to either

Reply:
yep you are right. I think now I've fixed it. Though I wonder, should I clean up this query?
Its a bit outdated and it returns different array depending if its combinational product or not. Or just leave it?